velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1488,4 +1488,15 @@ struct TimeZoneMinuteFunction : public TimestampWithTimezoneSupport<T> {
   }
 };
 
+template <typename T>
+struct ToISO8601Function {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<Varchar>& result,
+      const arg_type<Date>& date) {
+    result = DATE()->toString(date);
```

Comment:
Shouldn't this be DateType::toIso8601 ?

Reply:
You're right - sorry, looks like this got reverted in a previous commit. Thanks for catching that, fixed.